src/components/form-helpers/form-helpers.js

```diff
@@ -52,6 +52,46 @@ export const FieldItem = connect(
   ),
 );
 
+export const PasswordWrapper = connect(
+  ({ className, fieldName, children, formik: { errors, touched } }) => (
+    <li
+      className={concatClasses(className, touched[fieldName] && errors[fieldName] ? 'error' : '')}
+    >
+      {children}
+      <div className="wrapper-password">
+        <p className="password-message">
+          <span
+            className={
+              !touched[fieldName]
+                ? 'waiting-message'
```

Comment:
Is waiting status really necessary? I mean, I thought that the validation status was always visible.

Reply:
I verified:

Pristine:

![image](https://user-images.githubusercontent.com/1157864/55996187-070dcd80-5c8d-11e9-959b-33a668961e02.png)

Touched and cleaned:

![image](https://user-images.githubusercontent.com/1157864/55996165-f9f0de80-5c8c-11e9-9512-dd41c31270da.png)

--- 

In my opinion, for the circles, it should not care if the field has been touched or not, the red border is enough to show that there is an error. But it is just an opinion, you could talk it with Santiago or @IgnacioRodrigues

Reply:
By default is waiting state, in pristine state. I don't think they should be the same, the issue here is avoid showing errors prematurely to the user, imagine if you go to a form and it's all red and you didn't even typed a thing. O_O